tests/storage/outofprocess_test.py

```diff
@@ -967,61 +982,56 @@ def test_glob(oop_cleanup, tmpdir):
 
 # External APIs
 
-def test_read_lines(oop_cleanup, tmpdir):
+def test_read_lines(oop_cleanup, user_file):
     iop = oop.getProcessPool("test")
-    path = str(tmpdir.join("file"))
 
-    with open(path, "wb") as f:
+    with open(user_file.path, "wb") as f:
         f.write(b"1\n2\n3\n")
-    assert iop.readLines(path) == [b"1", b"2", b"3"]
+    assert iop.readLines(user_file.path) == [b"1", b"2", b"3"]
 
 
-def test_write_lines(oop_cleanup, tmpdir):
+def test_write_lines(oop_cleanup, user_file):
     iop = oop.getProcessPool("test")
-    path = str(tmpdir.join("file"))
 
-    iop.writeLines(path, [b"1\n", b"2\n", b"3\n"])
-    with open(path, "rb") as f:
+    iop.writeLines(user_file.path, [b"1\n", b"2\n", b"3\n"])
+    with open(user_file.path, "rb") as f:
         assert f.read() == b"1\n2\n3\n"
```

Comment:
Previously we tested that io.writeLines() creates the file if the file does not exists. Now we use an existing file, so we don't test the behavior when file does not exist.

This test needs a directory, not a file, so we better use the "mount-512" and create a temporary directory inside this mount. See localfssd_test.py for example of using "mount-512" in a safe way.

Reply:
Done, see comment above. I will close all these threads and we can follow the discussion in https://github.com/oVirt/vdsm/pull/148#discussion_r857135030 .